main.nf

```diff
@@ -442,16 +419,11 @@ process soft_filter {
         tuple path("WI.${date}.soft-filter.vcf.gz"), path("WI.${date}.soft-filter.vcf.gz.csi"), emit: soft_filter_vcf
         path "WI.${date}.soft-filter.vcf.gz.tbi"
         path "WI.${date}.soft-filter.stats.txt", emit: 'soft_vcf_stats'
+        path "WI.${date}.soft-filter.filter_stats.txt"
 
-    script:
-        os='uname'.execute().text.strip() == "Darwin" ? "macos" : "linux"
-    """
-        function cleanup {
-            rm ad_dp.filtered.vcf.gz
-        }
-        trap cleanup EXIT
```

Comment:
I literally just got an email from Quest today that I'm taking up too much scratch space lol

I left it out since I was wondering whether we should add this to all processes in all pipelines that generate a large data. Or have you tried:
https://github.com/nextflow-io/nf-hack18/issues/3
https://github.com/nextflow-io/nextflow/blob/cli-docs/docs/cli.rst

When you get around let's discuss and I'll do a more systematic update. Currently alignment-nf takes up 3x more space than gatk-nf in working dir.

Reply:
Let’s discuss tomorrow